http/src/main/resources/init-resources/startup.sh

```diff
@@ -74,6 +74,25 @@ SERVER_CRT=$(proxyServerCrt)
 SERVER_KEY=$(proxyServerKey)
 ROOT_CA=$(rootCaPem)
 
+mkdir -p /mnt/disks/work
+chmod a+rwx /mnt/disks/work
```

Comment:
I think what happens is when runtime is rebooted, the directory is re-created, but permission sometimes is still not set properly. Hence, adding chmod here as well to reduce some flakiness